roles/systems/postgres/defaults/main.yaml

```diff
@@ -0,0 +1,2 @@
+postgres_server_ssl_cert_file: ""
+postgres_server_ssl_key_file: ""
```

Comment:
Not a fan of no line endings in files. This will pass, but in the future validate.

Reply:
Fixed this